panels/metrics/cc-metrics-panel.ui

```diff
@@ -5,100 +5,148 @@
-                    <property name="xalign">0</property>
-                    <attributes>
-                      <attribute name="weight" value="bold"/>
-                    </attributes>
-                  </object>
-                </child>
-                <child>
-                  <object class="GtkLabel" id="metrics_explanation_label">
-                    <property name="visible">1</property>
-                    <property name="margin-bottom">12</property>
-                    <property name="use-markup">1</property>
-                    <property name="wrap">1</property>
-                    <property name="max-width-chars">50</property>
-                    <property name="xalign">0</property>
-                    <property name="use_underline">1</property>
-                    <property name="label" translatable="yes">_Endless collects metrics on user behavior and actions. All data sent is anonymous. We use the data to improve the system.</property>
-                  </object>
-                </child>
-                <child>
-                  <object class="GtkListBox" id="metrics_list_box">
-                    <property name="visible">1</property>
-                    <property name="can-focus">1</property>
-                    <property name="selection-mode">none</property>
-                    <property name="margin_bottom">16</property>
-		    <child>
-                      <object class="CcListRow" id="metrics_identifier_row">
-                        <property name="visible">True</property>
-                        <property name="title" translatable="yes">Your Metrics Identifier</property>
-                      </object>
-                    </child>
-                  </object>
-                </child>
-
+              <object class="GtkImage">
+                <property name="visible">true</property>
+                <property name="valign">start</property>
+                <property name="pixel-size">96</property>
+                <!-- FIXME: use proper icon -->
+                <property name="icon-name">action-unavailable-symbolic</property>
+                <style>
+                  <class name="dim-label" />
+                </style>
+              </object>
+            </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="visible">true</property>
+                <property name="margin-top">20</property>
+                <property name="margin-bottom">15</property>
+                <property name="label" translatable="yes">Metrics services turned off</property>
+                <attributes>
+                  <attribute name="scale" value="1.44" />
+                </attributes>
+              </object>
+            </child>
+            <child>
+              <object class="GtkLabel">
+                <property name="visible">true</property>
+                <property name="label" translatable="yes">No applications can obtain metrics information.</property>
```

Comment:
This is not what turning metrics off means.

Reply:
```suggestion
                <property name="label" translatable="yes">No anonymous usage data will be sent to Endless.</property>
```

perhaps.

Reply:
Addressed